staging-service/src/test/java/com/github/cafdataprocessing/services/staging/dao/FileSystemDaoTest.java

```diff
@@ -55,7 +55,7 @@
     private static String TEST_TENANT_ID = "12345";
     private TenantId tenantId;
     private String baseDirName;
-    private String storageDirName = "/etc/store/batches";
+    private String storageDirName;
```

Comment:
I didn't understand the purpose of this change.

Reply:
Used to be null, and was failing tests with a NPE with these changes.